types/test.d.ts

```diff
@@ -997,6 +1011,12 @@ export type PlaywrightTestOptions = {
    */
   colorScheme: ColorScheme | undefined;
 
+  /**
```

Comment:
I thought those would go through as is, now I can see that we wrap each value :/

Reply:
Yeah :/ lets rework it in a follow-up.